virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,43 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
-
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+|*Volumes project* list
+|Select the volume to boot your VM from.
 
-// |*Add volume*
-// |Upload a new volume or use an existing PersistentVolumeClaim (PVC).
+|*Add volume*
+|Upload a new volume or use an existing PersistentVolumeClaim (PVC).
 
-// |*Filter* field
-// |Filter boot sources by operating system.
+|*Filter* field
+|Filter boot sources by operating system.
 
-// |Search field
-// |Search boot sources by keyword.
+|Search field
+|Search boot sources by keyword.
 
-// |*Select InstanceType*
-// |Select an instance type based on the number of CPUs and amount of memory available.
+|*Select InstanceType*
```

Comment:
"Select InstanceType" is text for this section. I think the UI element would be called "**InstanceType** cards". Definition would need to be updated accordingly but I'm not sure how.